app/assets/javascripts/analytics_toolkit/google-analytics-universal-tracker.js

```diff
@@ -3,11 +3,11 @@
 
   var $ = global.jQuery
   var GOVUK = global.GOVUK || {}
-  var pii = new GOVUK.pii()
-  pii.stripDatePII = true
-  pii.stripPostcodePII = true
+  var pii
 
   var GoogleAnalyticsUniversalTracker = function (trackingId, fieldsObject) {
+    pii = new GOVUK.pii()
```

Comment:
Is there a difference in initialising this variable here as opposed to where it was? I'm just curious, not suggesting it be changed.

Reply:
Yeah, when GoogleAnalyticsUniversalTracker is initialized, we want the latest version of the DOM. But, if pii is initialized in the global scope it has state determined from the original version of the DOM.

In practice it might not matter, since we don't tend to change meta tags on the fly, but it's helpful for tests.